src/app/map/page.tsx

```
@@ -1,34 +1,22 @@
-              className={iconWrapper({
-                size: 'lg',
-                radius: 'full',
-                border: 'black',
-                margin: 'xs',
-              })}
-            >
-              <category.icon />
-            </div>
-            <p className={categoryTitle()}>{category.label}</p>
-          </div>
+          <Link href={`/map/search?type=${category.value}`}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)